uber/static/angular-apps/signups/possible.html

```diff
@@ -40,3 +40,9 @@
     </td>
 </tr>
 </table>
+
+<div ng-show="job in jobs == 0 || job in jobs == null">
+    <p>
+        There are currently no shifts available. Please contact <a href="mailto:stops@magfest.org">stops@magfest.org</a> for more information.
+    </p>
```

Comment:
this is a great first pass, good stuff.

1) unless I'm misreading this, there might be an easier way to detect no jobs are present, maybe like something from here: http://stackoverflow.com/questions/23076570/ng-show-when-array-length-is-zero  that one with nh-hide seems like a good fit

2) something we're doing is trying to make everything in this repository be non-magfest specific, so that any event can use it.  could you replace ```stops@magfest.org``` with ```{{ c.STAFF_EMAIL }}```? (it might break the mailto href though, but that's ok)

in magfest's sideboard/plugins/uber/development.ini, it defines this line:
```
staff_email = "MAGFest Staffing <stops@magfest.org>"
```

3) One thing we're trying to communicate/hint to the user here is that, while shifts might exist in the system, there just might not be any shifts available that they're eligible to sign up for, and they should email our staffing crew to find out what the deal is.  

example: if a staffer is assigned to gameroom, but all the gameroom shifts are already signed up for by other people, this page won't show any shifts.  That would prompt our gameroom staff to email the staff folks or the gameroom folks and ask 'wtf why can't I sign up for shifts?'

So, we could change the wording here to be a bit more along the lines of "there are no shifts that you are eligible to sign up for", something like that.